packages/ndla-ui/src/Frontpage/FrontpageCircularSubject.tsx

```diff
@@ -84,13 +84,13 @@ type Props = {
   category: categoryProp;
 };
 
-const FrontpageCircularSubject: React.FunctionComponent<
-  Props & React.HTMLProps<HTMLButtonElement> & tType
-> = ({ category, t, ...rest }) => {
+const FrontpageCircularSubject: React.FunctionComponent<Props &
+  React.HTMLProps<HTMLButtonElement> &
+  tType> = ({ category, t, ...rest }) => {
   // @ts-ignore
   const Illustration = categoryIllustrations[category.name];
   return (
-    <StyledButton type="button" {...rest}>
+    <StyledButton type="button">
```

Comment:
Når eg prøver å sette denne tilbake til å ha med ...rest får eg feilmelding `Type 'string' is not assignable to type '"button" | "reset" | "submit" | undefined'.`. Skjønner ikkje heilt kvifor dette skjer.

Reply:
Men det er kanskje ikkje så farlig i og med at denne komponenten ikkje lenger er i bruk.